app/controllers/api/profiles_controller.rb

```diff
@@ -16,6 +16,7 @@ def update
       profile = person.profiles.find(params.require(:id))
 
       profile.update!(profile_attributes)
+      Notifier.prepare_notifications(topic: profile.person, action_name: 'update')
```

Comment:
should we always call this?
You can call the service even without passing in new assessment answers

Reply:
@alexdesi good question; yes I think we should. Its important to notify the suppliers of _any_ changes.